espnet2/gan_svs/espnet_model.py

```diff
@@ -333,6 +340,8 @@ def collect_feats(
         duration_ruled_phn_lengths: Optional[torch.Tensor] = None,
         duration_syb: Optional[torch.Tensor] = None,
         duration_syb_lengths: Optional[torch.Tensor] = None,
+        slur: Optional[torch.Tensor] = None,
```

Comment:
I guess you also need to add slur preparation in the TEMPLATE/svs1/svs.sh and data.sh ?